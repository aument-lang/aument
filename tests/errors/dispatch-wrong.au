struct Class {
}

func (self: Class) init() {
}

init("A");