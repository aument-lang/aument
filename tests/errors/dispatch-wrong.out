interpreter error(2) in -: incompatible call
7 | init("A");
